sceptre/resolvers/stack_output.py

```diff
@@ -170,10 +180,24 @@ def resolve(self):
 
         stack_argument = arguments[0]
         if len(arguments) > 1:
-            extra_args = arguments[1].split("::", 2)
-            profile, region, sceptre_role = extra_args + (3 - len(extra_args)) * [None]
+            try:
+                extra_args = arguments[1].split("::", 2)
+                profile, region, sceptre_role = extra_args + (3 - len(extra_args)) * [
+                    None
+                ]
```

Comment:
Not related to my change but I find this expression confusing and would propose to rewrite it:
```python
if len(extra_args) > 0:
    profile = extra_args[0]
if len(extra_args) > 1:
    region = extra_args[1]
if len(extra_args) > 2:
    sceptre_role = extra_args[2]
```

Reply:
Maybe even
```
if len(extra_args) > 0:
    profile = extra_args[0]
if len(extra_args) > 1:
    region = extra_args[1]
if len(extra_args) > 2:
    sceptre_role = extra_args[2]
if len(extra_args) > 3:
    raise ValueError
```

Reply:
Yes, please. The original expression looks like C code.